### ISSUE TITLE
Semilla

### ISSUE THREAD

**Issue by Tulmot** [Tulmot/Sklearn-Multilabel#20]
Hacer una semilla para usar luego los conjuntos de datos

**Comment by Tulmot** [comment]
He encontrado la función random.seed(), y mi duda es si vale crear matrices aleatorias (con mismo tamaño) y luego las voy asignando a la semilla 0,1,2.....
Porque he estado mirando como calcula la semilla en bagging y no lo comprendo muy bien, además que esa clase recibe parámetros que yo no tengo y les usa para calcular la semilla.

**Comment by alvarag** [comment]
Perdona, no te había leído esta issue. Supongo que lo habrás solucionado pero aquí tienes la explicación de por qué utilizar *check_random_state*
http://scikit-learn.org/stable/developers/utilities.html

De este modo recibes un random_state en el constructor y en el método fit haces la llamada a *check_random_state*, tal y como hace Bagging (por ejemplo):

 ```
random_state = check_random_state(self.random_state)
```
